packages/main/src/DateComponentBase.js

```diff
@@ -26,6 +26,17 @@ const metadata = {
 			type: CalendarType,
 		},
 
+		/**
+		 * Sets a secound calendar type used for display.
```

Comment:
Defines the secondary calendar type
If not set, the calendar will only show the primary calendar type

add since tag as well

Reply:
Done